controllers/upgrade_handlers.go

```diff
@@ -169,6 +174,14 @@ func (r *ImageBasedUpgradeReconciler) prePivot(ctx context.Context, ibu *lcav1al
 	return doNotRequeue(), nil
 }
 
+func (r *ImageBasedUpgradeReconciler) getDesiredStaterootRepo(ibu *lcav1alpha1.ImageBasedUpgrade) string {
+	return fmt.Sprintf("/host/ostree/deploy/rhcos_%s/var", ibu.Spec.SeedImageRef.Version)
+}
+
+func (r *ImageBasedUpgradeReconciler) remountSysroot() (string, error) {
```

Comment:
we already have a function for this in ops.

Reply:
Updated